src/hash.rs

```diff
@@ -71,16 +71,21 @@ fn hash_algorithm<B: AsRef<[u8]>>(name: &str, bytes: B) -> Result<String> {
             hasher.write(bytes.as_ref());
             Ok(format!("{:x}", hasher.finish()))
         }
+        "xxh64_fixed" => {
+            let mut hasher = XxHash64::with_seed(17479268743136991876);
```

Comment:
seed?

Reply:
Yes. It's a fixed hash, so the seed needs to be the same for all hashes. This is used for caching checks in iconforge

Reply:
why this number

Reply:
I picked it. It doesn't have any meaning.

Reply:
```suggestion
            let mut hasher = XxHash64::with_seed(17479268743136991876); // random number
```